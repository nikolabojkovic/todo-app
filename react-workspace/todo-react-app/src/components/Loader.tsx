export function Loader() {
  return (<div className='loader-container d-flex justify-content-center align-items-center fade-in'>
            <div className='loader'></div>
          </div>);
}