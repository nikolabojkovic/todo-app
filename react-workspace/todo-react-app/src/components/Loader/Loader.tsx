type Props = {
  height: number;
}

export function Loader({ height }: Props) {
  return (<div id="loader" data-testid="loader" style={{height: height + 'px'}} className='loader-container d-flex justify-content-center align-items-center fade-in'>
            <div className='loader'></div>
          </div>);
}