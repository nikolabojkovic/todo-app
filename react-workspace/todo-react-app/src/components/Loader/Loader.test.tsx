import renderer from 'react-test-renderer';
import { Loader } from './Loader';

describe('ImportExport', () => {
  it('component should match snapshot', () => {
    const jsxElement = <Loader height={280} />;
    const tree = renderer.create(jsxElement).toJSON();
    expect(tree).toMatchSnapshot();
  });
});