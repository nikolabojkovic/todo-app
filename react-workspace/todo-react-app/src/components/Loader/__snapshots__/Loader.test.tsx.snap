// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImportExport component should match snapshot 1`] = `
<div
  className="loader-container d-flex justify-content-center align-items-center fade-in"
  data-testid="loader"
  id="loader"
  style={
    Object {
      "height": "280px",
    }
  }
>
  <div
    className="loader"
  />
</div>
`;
